grasa_event_locator/settings/base.py

```diff
@@ -18,7 +18,7 @@
 
 # CONFIGURATION
 # Load configuration from disk
-config = yaml.safe_load(os.environ.get("CONFIGPATH", open("config.yml")))
+CONFIG = yaml.safe_load(os.environ.get("CONFIGPATH", open("config.yml")))
```

Comment:
I appreciate your changing the variable name to be in all caps to conform with coding standards.